activiti-cloud-services-modeling/activiti-cloud-services-modeling-service/src/main/resources/schema/connector-schema.json

```diff
@@ -24,7 +24,7 @@
           },
           "type": {
             "type": "string",
-            "enum": [ "string", "integer", "boolean", "date", "json", "file" ]
+            "enum": [ "string", "integer", "boolean", "date", "json", "file", "datetime", "string-list" ]
```

Comment:
Just to help other people who may review this PR, `string-list` is not a variable type but an input type. 

We are defining here how a form to set the input variables for a connector event will be render and in the FE has been added a new input type for facilitating the user the setting of a string list